packages/swap/src/routes/swap.ts

```diff
@@ -129,6 +129,7 @@ router.get(
       srcAsset,
       destAsset,
       destAddress: readField('destAddress'),
+      createdAt: swapDepositChannel?.createdAt.valueOf(),
```

Comment:
might be a bit confusing as its not defined for smart contract calls
```suggestion
      depositAddressCreatedAt: swapDepositChannel?.createdAt.valueOf(),
```

Reply:
good point 🤦